### PR TITLE
Signinscreen

### PR DESCRIPTION
**Pull request by jamiegoh**
Hello besties, this is just the sign in screen with just the button that redirects to map <3

### CODE REVIEW

**Comment by RealDyllon** [comment]
@jamiegoh navigation needs to be replaced with the new go_router dependency.

**Comment by RealDyllon** [comment]
@jamiegoh any update on this?

**Comment by jamiegoh** [comment]
@RealDyllon i added the go router thingy

File: pubspec.yaml [hunk 1]
```diff
@@ -67,8 +67,8 @@ flutter:
   uses-material-design: true
 
   # To add assets to your application, add an assets section, like this:
-  # assets:
-  #   - images/a_dot_burr.jpeg
+  assets:
+    - assets/singpass.png
   #   - images/a_dot_ham.jpeg
```
Comment (RealDyllon) [inline pubspec.yaml]: remove this

File: README.md [hunk 2]
```diff
@@ -1 +1,2 @@
-# aura-app
\ No newline at end of file
+# aura-app 
+#test
```
Comment (RealDyllon) [inline README.md]: remove this

File: lib/view/signin/signin_screen.dart [hunk 3]
```diff
@@ -0,0 +1,30 @@
+import 'package:flutter/material.dart';
```
Comment (RealDyllon) [inline lib/view/signin/signin_screen.dart:1]: good

File: lib/router.dart [hunk 4]
```diff
@@ -66,6 +67,10 @@ final router = GoRouter(
                   const ChangeHomeAddressScreen())
         ],
       ),
+      GoRoute(
+          path: "/sign-in",
+          builder: (BuildContext context, GoRouterState state) =>
+              const SigninScreen())
```
Comment (RealDyllon) [inline lib/router.dart:73]: good stuff

File: pubspec.yaml [hunk 5]
```diff
@@ -67,8 +67,8 @@ flutter:
   uses-material-design: true
 
   # To add assets to your application, add an assets section, like this:
-  # assets:
-  #   - images/a_dot_burr.jpeg
+  assets:
+    - assets/singpass.png
```
Comment (RealDyllon) [inline pubspec.yaml:71]: genius 5/5 👍🏼

**Review by RealDyllon — CHANGES_REQUESTED** [review]

**Comment by jamiegoh** [comment]
@RealDyllon fixed bestie

**Review by RealDyllon — APPROVED** [review]
LGTM great stuff jamie keep it up

**Pull request merged**